superset-frontend/src/SqlLab/components/SqlEditor/index.tsx

```diff
@@ -940,12 +945,14 @@ const SqlEditor: FC<Props> = ({
           minWidth={SQL_EDITOR_LEFTBAR_WIDTH}
           initialWidth={SQL_EDITOR_LEFTBAR_WIDTH}
           enable={!hideLeftBar}
+          padding={leftBarPadding}
```

Comment:
@justinpark Why does a left bar padding influences on a wrapper height? If it was a top/bottom padding, I would understand but a left one?

Reply:
If this variable mean that a top padding is applied when the left bar is open, then it would be a good idea to rename the variable to make it more clear.